nextflow_schema.json

```diff
@@ -10,7 +10,10 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": ["input", "outdir"],
+            "required": [
+                "input",
+                "outdir"
+            ],
```

Comment:
Im not sure, but i think in this format, `prettier` still needs to be run on the schema.

Reply:
Will do!